linked_list/linked_list.py

```diff
@@ -1,75 +1,138 @@
-
 # Defines a node in the singly linked list
 class Node:
 
-    def __init__(self, value, next_node = None):
+    def __init__(self, value, next_Node = None):
         self.value = value
-        self.next = next_node
+        self.next = next_Node
+        
 
 # Defines the singly linked list
 class LinkedList:
     def __init__(self):
-      self.head = None # keep the head private. Not accessible outside this class
+        self.head = None # keep the head private. Not accessible outside this class
+        self.tail = None
 
     # returns the value in the first node
     # returns None if the list is empty
     # Time Complexity: ?
     # Space Complexity: ?
     def get_first(self):
-        pass
+        if self.head is None:
+            return None
+        
+        return self.head.value
 
 
     # method to add a new node with the specific data value in the linked list
     # insert the new node at the beginning of the linked list
     # Time Complexity: ?
     # Space Complexity: ?
     def add_first(self, value):
-        pass
+        if self.head is None:
+            self.head = self.tail = Node(value)
+        else:
+            new_Node = Node(value, self.head)

```

Comment:
Remember our Python style conventions!
```suggestion
            new_node = Node(value, self.head)

```